gulp/sass.js

```diff
@@ -28,3 +28,28 @@ gulp.task('sass', function () {
     .pipe(sourcemaps.write())
     .pipe(gulp.dest(config.paths.public + '/stylesheets/'))
 })
+
+gulp.task('sass-documentation', function () {
+  return gulp.src(config.paths.docsAssets + '/sass/*.scss')
+    .pipe(sourcemaps.init())
+    .pipe(sass({ outputStyle: 'expanded' }).on('error', sass.logError))
+    .pipe(sourcemaps.write())
+    .pipe(gulp.dest(config.paths.public + '/stylesheets/'))
+})
+
+// Backward compatibility with Elements
+
+gulp.task('sass-v6', function () {
+  return gulp.src(config.paths.v6Assets + '/sass/*.scss')
+    .pipe(sourcemaps.init())
+    .pipe(sass({
+      outputStyle: 'expanded',
+      includePaths: [
+        'node_modules/govuk_frontend_toolkit/stylesheets',
+        'node_modules/govuk-elements-sass/public/sass',
+        'node_modules/govuk_template_jinja/assets/stylesheets'
+      ]
+    }).on('error', sass.logError))
+    .pipe(sourcemaps.write())
+    .pipe(gulp.dest(config.paths.public + '/v6/stylesheets/'))
+})
```

Comment:
Don’t need these tasks as we no longer have the documentation folder, or need v6 support